contracts/contracts/staking/SingleAssetStaking.sol

```diff
@@ -37,6 +37,7 @@ contract SingleAssetStaking is Initializable, Governable {
     uint256[] public rates; // rates that correspond with the allowed durations
 
     uint256 public totalOutstanding;
+    uint256 public totalStakers;
```

Comment:
Contract is already deployed and uses a proxy. So I believe any new slot should be appended, not inserted in the middle? Move totalStakers after bool paused.

Reply:
Nit: add a comment explaining this is the total number of accounts that have at least one active stake (e.g. unpaid)